src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -775,13 +781,16 @@ def _get_custom_keys(cls, data: Dict):
             module = importlib.import_module(m)
             cls = getattr(module, custom_cls)
         except ImportError:
-            raise ValueError(
+            error = ValueError(
```

Comment:
why wrap? same other places

Reply:
If not wrapped, the `error category` will be `UNKNOW`, and `the error` type is Value Error, we need to solve the problem of the `error category` being `UNKNOW`.
This modification will set the `error category` to `UserError `and the `error type` to `ValueError`.
![image](https://github.com/microsoft/promptflow/assets/23182548/334bd39f-179d-47e2-af1c-8f19fd486459)

Reply:
I think we could change the ErrorType? May make a new Error class if needed

Reply:
ok, according to offline discussions, we will further optimize the details for the next PR, and merge the current PR first.